internal/deb/extract_test.go

```diff
@@ -352,6 +352,40 @@ var extractTests = []extractTest{{
 		},
 	},
 	error: `cannot extract from package "test-package": path /dir/ requested twice with diverging mode: 0777 != 0000`,
+}, {
+	summary: "Hard link must be created if specified in the tarball",
```

Comment:
Nit: I think something like "Hard link is extracted" is shorter and has the same meaning.

Reply:
Fixed in [ffd9951](https://github.com/letFunny/chisel/pull/17/commits/ffd99512f315c63893ae49055f3ad630d6238dd6)